diff --git a/dist/vite.mjs b/dist/vite.mjs
index 2fc59cfb4d9d29a52c17164b8ad9c30413046ad9..ff62c452500ceb6eb62952e06e80a44fcdb11c5b 100644
--- a/dist/vite.mjs
+++ b/dist/vite.mjs
@@ -257,7 +257,7 @@ function ConfigHMRPlugin({ ready }) {
 function UnocssSvelteScopedVite(options = {}) {
   const context = createSvelteScopedContext(options.configOrPath);
   if (context.uno.config.transformers)
-    throw new Error('Due to the differences in normal UnoCSS global usage and Svelte Scoped usage, "config.transformers" will be ignored. You can still use transformers in CSS files with the "cssFileTransformers" option.');
+    console.warn('Due to the differences in normal UnoCSS global usage and Svelte Scoped usage, "config.transformers" will be ignored. You can still use transformers in CSS files with the "cssFileTransformers" option.');
   if (!options.classPrefix)
     options.classPrefix = "uno-";
   const plugins = [
